packages/web-components/src/components/masthead/masthead-contact.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { html, customElement, property } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings.js';
+import Chat20 from 'carbon-web-components/es/icons/chat/20.js';
+import ddsSettings from '../../internal/vendor/@carbon/ibmdotcom-utilities/utilities/settings/settings';
+import styles from './masthead.scss';
+import DDSMastheadProfile from './masthead-profile';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The contact button UI in the masthead.
+ *
+ * @element dds-masthead-contact
+ */
+@customElement(`${ddsPrefix}-masthead-contact`)
+class DDSMastheadContact extends DDSMastheadProfile {
+  /**
+   * The `aria-label` attribute for the trigger button.
+   */
+  @property({ attribute: 'trigger-label' })
+  triggerLabel = 'Contact';
+
+  render() {
+    const { triggerLabel, _handleClick: handleClick } = this;
+    return html`
+      <a
+        tabindex="0"
```

Comment:
I realize this is probably copied from the `DDSMastheadProfile` render method, but do we need `tabindex` on an `<a>`?